contracts/Comptroller.sol

```diff
@@ -1081,26 +1081,9 @@ contract Comptroller is ComptrollerV3Storage, ComptrollerInterface, ComptrollerE
         return state;
     }
 
-    function _become(Unitroller unitroller, uint compRate_, address[] memory compMarketsToAdd, address[] memory otherMarketsToAdd) public {
+    function _become(Unitroller unitroller) public {
```

Comment:
All of this logic related to initialization can be stripped out since it's already occurred, correct? It's not missing any original safety checks?

Reply:
Yes exactly, and we were worried that having to specify those args such that they had no effect would make it harder for the proposer at 'runtime' (i.e. same comp rate, no new comped markets, and no new other markets), so this should decrease the surface area for proposal mistakes.

Reply:
The other way to think about this is a G4 Comptroller, which, in this case, does not need any initialization.